resources/frontend/frontend-config.yaml

```
@@ -3,4 +3,4 @@ kind: ConfigMap
 metadata:
   name: showcase-frontend-config
 data:
-  config.json: '{"API_URL": "https://backend.{YOUR_CLUSTER_DOMAIN}/v1/images"}'
+  config.json: '{"API_URL": "https://backend.{YOUR_CLUSTER_DOMAIN}/v1/images","API_KEY":"&key={YOUR_MAP_API_KEY}"}'
```

It would be better to have the only the key here, not key as a query parameter.
```suggestion
  config.json: '{"API_URL": "https://backend.{YOUR_CLUSTER_DOMAIN}/v1/images","API_KEY":"{YOUR_MAP_API_KEY}"}'
```

---

Or change the name here to API_KEY_PARAM
```suggestion
  config.json: '{"API_URL": "https://backend.{YOUR_CLUSTER_DOMAIN}/v1/images","API_KEY_PARAM":"&key={YOUR_MAP_API_KEY}"}'
```